<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>node</class>
 <widget class="QWidget" name="node">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_node_tree" native="true">
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTreeView" name="treeView"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_node_panel" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>30</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>节点名称：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lb_nodeName">
          <property name="text">
           <string>0bca</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>上一级根节点：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lb_parNodeName">
          <property name="text">
           <string>ffff</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="checkBox_temp">
          <property name="text">
           <string>温度：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lb_temp">
          <property name="text">
           <string>28</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QCheckBox" name="checkBox_light">
          <property name="text">
           <string>灯泡：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lb_lightStatus">
          <property name="text">
           <string>未开启</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_opLight">
          <property name="text">
           <string>打开</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_clLight">
          <property name="text">
           <string>关闭</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QCheckBox" name="checkBox_fan">
          <property name="text">
           <string>风扇：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lb_fanStatus">
          <property name="text">
           <string>已开启</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_opFan">
          <property name="text">
           <string>打开</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_clFan">
          <property name="text">
           <string>关闭</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
